docs/prem-app/troubleshooting/service-download-hanging.md

```diff
@@ -0,0 +1,24 @@
+---
+id: service-download-hanging
+title: Service Download Hanging
+sidebar_label: Service Download Hanging
+sidebar_position: 1
+---
+
+# Service Download Hanging
```

Comment:
```suggestion
# Prem Server

## Service Download Hanging
```

Reply:
If we accept these changes, then we need to rename the file too.

Reply:
The idea is to have one sidebar element for each issue or one page for all?

Reply:
I was thinking of one sidebar element for each issue. Easier to read and find. What do you think?